mmdeploy/codebase/__init__.py

```diff
@@ -1,16 +1,21 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-import importlib
+from typing import List
 
 from mmdeploy.utils import Codebase
 from .base import BaseTask, MMCodebase, get_codebase_class
 
-extra_dependent_library = {
-    Codebase.MMOCR: ['mmdet'],
-    Codebase.MMROTATE: ['mmdet']
-}
 
+def import_codebase_rewriter(codebase_type: Codebase,
```

Comment:
Why do we add another function?
`import_codebase_rewriter` and `import_codebase` always works together.